client/src/sass/_beatmap-result.scss

```diff
@@ -1,5 +1,5 @@
 $margin: 14px;
-$image-size: 180px;
+$image-size: 190px;
```

Comment:
Why was this changed?

Reply:
It was a hack to give more space around the last `Statistic` item.  I'll revert the changes to this file, but can you give me some guidance on how you'd like to handle this?